src/rules/newline-after-import.js

```diff
@@ -42,9 +42,12 @@ function isClassWithDecorator(node) {
   return node.type === 'ClassDeclaration' && node.decorators && node.decorators.length
 }
 
+const EXPECTED_LINE_DIFFERENCE = 2
+
 module.exports = {
   meta: {
     docs: {},
+    fixable: 'code',
```

Comment:
Similar [`newline-after-var` rule](https://github.com/eslint/eslint/blob/master/lib/rules/newline-after-var.js#L26) in eslint uses the `whitespace` option. I haven't found the difference between two options in the docs, but just for consistency I'd suggest to use `whitespace`.